packages/lib/gpcircuits/circuits/proto-pod-gpc.circom

```diff
@@ -140,13 +151,44 @@ template ProtoPODGPC (
         isNullfierHashRevealed <== ownerIsNullfierHashRevealed
     );
 
+    /*
+     * 1 TupleModule with its inputs & outputs.
+     */
+
+    signal input tupleIndices[MAX_TUPLES][ARITY];
+    
+    signal tupleHashes[MAX_TUPLES] <== TupleModule(ARITY, MAX_ENTRIES, MAX_TUPLES)(entryValueHashes, tupleIndices);
+
+    /*
+     * 1 ListMembershipModule with its inputs & outputs.
+     */
+
+    // Index of entry value (if less than `MAX_ENTRIES`) or tuple hash that ought to be a member.
```

Comment:
This scheme needs more description here.
Alternately if the use of this index beyond entries becomes a reusable thing, maybe there should be a block comment somewhere to explain how we lay out the list of possible inputs to further modules, including the special role of -1, the appending of entries, then tuples, then more stuff later.